test/de/de_time_units_later.test.ts

```diff
@@ -0,0 +1,151 @@
+        expect(result.start).not.toBeNull();
+        expect(result.start.get("year")).toBe(2012);
+        expect(result.start.get("month")).toBe(8);
+        expect(result.start.get("day")).toBe(20);
+
+        expect(result.index).toBe(0);
+        expect(result.text).toBe("in 10 Tagen");
+
+        expect(result.start).toBeDate(new Date(2012, 8 - 1, 20));
+    });
+
+    testSingleCase(chrono.de, "in 15 Minuten", new Date(2012, 7, 10, 12, 14), (result) => {
+        expect(result.index).toBe(0);
+        expect(result.text).toBe("in 15 Minuten");
+        expect(result.start.get("hour")).toBe(12);
+        expect(result.start.get("minute")).toBe(29);
+        expect(result.start.get("meridiem")).toBe(Meridiem.PM);
+
+        expect(result.start).toBeDate(new Date(2012, 7, 10, 12, 29));
+    });
+
+    testSingleCase(chrono.de, "nach 15 Minuten", new Date(2012, 7, 10, 12, 14), (result) => {
+        expect(result.index).toBe(0);
+        expect(result.text).toBe("nach 15 Minuten");
+        expect(result.start.get("hour")).toBe(12);
+        expect(result.start.get("minute")).toBe(29);
+        expect(result.start.get("meridiem")).toBe(Meridiem.PM);
+
+        expect(result.start).toBeDate(new Date(2012, 7, 10, 12, 29));
+    });
+
+    testSingleCase(chrono.de, "ab jetzt in drei Sekunden geht es los", new Date(2012, 7, 10, 12, 14), (result) => {
+        expect(result.index).toBe(0);
+        expect(result.text).toBe("ab jetzt in drei Sekunden");
+        expect(result.start.get("hour")).toBe(12);
+        expect(result.start.get("minute")).toBe(14);
+        expect(result.start.get("second")).toBe(3);
+
+        expect(result.start).toBeDate(new Date(2012, 7, 10, 12, 14, 3));
+    });
+
+    testSingleCase(chrono.de, "eine Minute ab jetzt", new Date(2012, 7, 10, 12, 14), (result) => {
+        expect(result.index).toBe(0);
+        expect(result.text).toBe("eine Minute ab jetzt");
+        expect(result.start.get("hour")).toBe(12);
+        expect(result.start.get("minute")).toBe(15);
+
+        expect(result.start).toBeDate(new Date(2012, 7, 10, 12, 15));
+    });
+
+    testSingleCase(chrono.de, "in 1 Stunde", new Date(2012, 7, 10, 12, 14), (result) => {
+        expect(result.index).toBe(0);
+        expect(result.text).toBe("in 1 Stunde");
+        expect(result.start.get("hour")).toBe(13);
+        expect(result.start.get("minute")).toBe(14);
+
+        expect(result.start).toBeDate(new Date(2012, 7, 10, 13, 14));
+    });
+
+    testSingleCase(chrono.de, "in 1,5 Stunden", new Date(2012, 7, 10, 12, 40), (result) => {
+        expect(result.index).toBe(0);
+        expect(result.text).toBe("in 1,5 Stunden");
+        expect(result.start.get("hour")).toBe(14);
+        expect(result.start.get("minute")).toBe(10);
+
+        expect(result.start).toBeDate(new Date(2012, 7, 10, 14, 10));
+    });
+});
+
+// test("Test - Strict mode", function () {
```

Comment:
Please don't leave the commented out code like this.
If we don't have DE strict mode, let's remove it.